### DISCUSSION TITLE
Is there a way to alway show tab number ?

### DISCUSSION THREAD

**Discussion by Rem0ld** [kovidgoyal/kitty#6208] [Q&A]
I modified tab display to show tab index, but when I rename, launch a command or cd somewhere I lose the tab number 
why?
Can I specify somewhere I want the number no matter what?

**Comment by kovidgoyal** [comment]
Change your tab title template to include the number

**Comment by Rem0ld** [comment]
```
tab_title_template                    "{fmt.fg.default}{index}:{title.split(':')[1]}"
active_tab_title_template       "{fmt.fg.active_tab_foreground}{fmt.bg.default}{index}:{title}"
```
Those are my settings but I'm using a customized tabar as well and I noticed some errors so I will need to further investigate into them to understand why I dont' have the index.

Thanks for the reply
